### PR TITLE
test: support multi version node cases

### PR DESCRIPTION
**Pull request by fs-eire**
This change update script tools/build.ts to copy v7-v10 node test cases and allow different node test version for test runner.

BrowserStack builds requires #120 as dependency to pass

### CODE REVIEW

File: tools/build.ts [hunk 1]
```diff
@@ -80,6 +81,51 @@ logger.info('Build.SubModules', '(2/2) Fetching submodules... DONE');
...
+  if (cleanInstall) {
+    fs.emptyDirSync(TEST_DATA_NODE);
+  }
+  logger.info('Build.TestData', `(1/3) Cleaning folder... ${cleanInstall ? 'DONE' : 'SKIPPED'}`);
+  // Step 2: copy node tests for different version
+  logger.info('Build.TestData', '(2/3) Copy tests...');
+  [['v7', '5af210ca8a1c73aa6bae8754c9346ec54d0a756e'],   // rel-1.2.3
+   ['v8', '5cc146270945f0d007b140fb59a892a60ba69f49'],   // rel-1.3.0
+   ['v9', '4e67414849122d3df78bd72cee5717f90e715d12'],   // rel-1.4.1
+   ['v10', 'b22041c3f16cf8bcca9ed93982d6ffdf6ebf3746'],  // master
```
Comment (hariharans29) [inline tools/build.ts]: Is the general idea to freeze a commit id after every onnx release ? For example after onnx locks down v10, we freeze the commit id for that and we keep rolling forward the commit id of master (for v11, and so on) ?

File: tools/test-runner-cli.ts [hunk 2]
```diff
@@ -35,11 +37,14 @@ const json = stripJsonComments(jsonWithComments, {whitespace: true});
 const whitelist = JSON.parse(json) as Test.WhiteList;
 logger.verbose('TestRunnerCli.Init', `Loading whitelist... DONE`);
 
-const DEFAULT_BACKENDS: ReadonlyArray<TestRunnerCliArgs.Backend> = ['cpu', 'wasm', 'webgl'];
+// The default backends and opset version lists. Those will be used in suite tests.
+const DEFAULT_BACKENDS: ReadonlyArray<TestRunnerCliArgs.Backend> =
+    args.env === 'node' ? ['cpu', 'wasm'] : ['cpu', 'wasm', 'webgl'];
+const DEFAULT_OPSET_VERSIONS: ReadonlyArray<number> = [10, 9, 8, 7];
```
Comment (hariharans29) [inline tools/test-runner-cli.ts]: Do we need to keep upgrading this after each onnx release too ? If so, I think we better document this...

File: tools/build.ts (hunk 1 above)
Comment (fs-eire) [reply tools/build.ts]: yes that is the idea.

File: tools/test-runner-cli.ts (hunk 2 above)
Comment (fs-eire) [reply tools/test-runner-cli.ts]: the answer is 'yes' ideally. but considering the fact that node test cases are rarely modified ( usually the change is to add cases to that folder), we can do this as demand.

there is another option: only support the stable ONNX version. This means we should only support v10 after ONNX v1.5 release (which will happen very soon though). thus we only update when a ONNX release happen

**Pull request merged**